streams/ae.m3u

```diff
@@ -39,6 +41,8 @@ https://svs.itworkscdn.net/alwoustalive/alwoustatv.smil/playlist.m3u8
 https://ikomg1.s.llnwi.net/alyaumtv/playlist.m3u8
 #EXTINF:-1 tvg-id="CartoonNetworkArabic.ae" status="online",Cartoon Network Arabia (1080p)
 https://shls-cartoon-net-prod-dub.shahid.net/out/v1/dc4aa87372374325a66be458f29eab0f/index.m3u8
+#EXTINF:-1 tvg-id="CitrussTV.ae" status="blocked",Citruss TV (720p) [Geo-blocked]
+https://d7km4gkevcbok.cloudfront.net/live/watch_720p30.m3u8
```

Comment:
could also be https://d7km4gkevcbok.cloudfront.net/live/watch.m3u8 though it's geo-blocked from my location.

Reply:
I tried a few. They return 404